ffcv/transforms/__init__.py

```diff
@@ -9,6 +9,7 @@
 from .translate import RandomTranslate
 from .mixup import ImageMixup, LabelMixup, MixupToOneHot
 from .module import ModuleWrapper
+from .randaugment import RandAugment
 
 __all__ = ['ToTensor', 'ToDevice',
            'ToTorchImage', 'NormalizeImage',
```

Comment:
RandAugment should be added to this list.
Further having the individual augmentations (rotate, contrast, ...) available would also be useful.